examples/tag/query/README.md

```diff
@@ -0,0 +1,83 @@
+Tag Query Example
+=================
+
+This is a simple example console application demonstrating how to use the
+SystemLink Tag services API to query for tags by property and output metadata
```

Comment:
instead of "output" could we say "return" or "retrieve"?

Reply:
Switching to the same wording as the comment for the other README.